addedhelpful info (#6366)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md

![CDP Share Permissions](images/aadj/cdp-share-permissions.png)
9. In the **Advanced Sharing** dialog box, click **OK**.

> > [!Tip]

Reviewer: ```suggestion
> [!TIP]
```

> > Make sure users can access **\\Server FQDN\sharename**.

Reviewer: ```suggestion
> Make sure that users can access **\\Server FQDN\sharename**.
```

Reviewer 2: Yes, that extra backslash as an escape character is necessary here.

> > Make sure that users can access **\\Server FQDN\sharename**.

Reviewer 2: ```suggestion
> Make sure that users can access **\\\Server FQDN\sharename**.
```

14. Click **Save**
15. Sign-out of the Azure portal.

> [!IMPORTANT]

> > For more details about the actual experience after everything is configured, please visit [this article](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-how-it-works-authentication).

Reviewer: ```suggestion
> For more details about the actual experience after everything has been configured, please see [Windows Hello for Business and Authentication](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-how-it-works-authentication).
```
